elfcore.test.sh/elf-sans-immobile.test.sh is not passing on Android-x86,
so it is disabled by this. These tests also fail on ARM, but are
not enabled by upstream on any targets except 64-bit x86.
The exact reason why the test fails, or what steps might be necessary
to fix it, are not known, but one way of describing the probable
effects that this test failing implies would be:
"the 'elfination' feature is most likely not working on Android"

--- a/tests/elf-sans-immobile.test.sh
+++ b/tests/elf-sans-immobile.test.sh
@@ -16,7 +16,7 @@
 . ./subr.sh
 
 run_sbcl <<EOF
-  #+(and linux x86-64 sb-thread)
+  #+(and linux x86-64 sb-thread (not android))
   (unless (member :immobile-space sb-impl:+internal-features+)
     (exit :code 0)) ; proceed with test
  (exit :code 2) ; otherwise skip the test
--- a/tests/elfcore.test.sh
+++ b/tests/elfcore.test.sh
@@ -16,7 +16,7 @@
 . ./subr.sh
 
 run_sbcl <<EOF
-  #+(and linux elf sb-thread)
+  #+(and linux elf sb-thread (not android))
   (let ((s (find-symbol "IMMOBILE-SPACE-OBJ-P" "SB-KERNEL")))
     (when (and s (funcall s #'car)) (exit :code 0))) ; good
  (exit :code 2) ; otherwise
